error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/util/ConflictDetectionTest.java

```diff
@@ -0,0 +1,62 @@
+package tech.picnic.errorprone.bugpatterns.util;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.MethodTreeMatcher;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.MethodTree;
+import org.junit.jupiter.api.Test;
+
+final class ConflictDetectionTest {
+  @Test
+  void matcher() {
+    CompilationTestHelper.newInstance(RenameBlockerFlagger.class, getClass())
+        .addSourceLines(
+            "/A.java",
+            "import static staticimport.B.foo3t;",
+            "",
+            "class A {",
+            "  private void foo1() {",
+            "    foo3t();",
+            "  }",
+            "",
+            "  // BUG: Diagnostic contains: [RenameBlockerFlagger] a method named `foo2t` is already defined in",
+            "  // this class or a supertype",
+            "  private void foo2() {}",
+            "",
+            "  private void foo2t() {}",
+            "",
+            "  // BUG: Diagnostic contains: [RenameBlockerFlagger] `foo3t` is already statically imported",
+            "  private void foo3() {}",
+            "",
+            "  // BUG: Diagnostic contains: [RenameBlockerFlagger] `int` is not a valid identifier",
+            "  private void in() {}",
+            "}")
+        .addSourceLines(
+            "/staticimport/B.java",
```

Comment:
```suggestion
            "/pkg/B.java",
```
In other classes we use `pkg` for this :).